lib/commands/performance.js

```diff
@@ -61,6 +73,59 @@ commands.getPerformanceData = async function getPerformanceData (packageName, da
   return data;
 };
 
+// API level 30 and above
+//   ['<System Type>', '<Memory Type>', <pss total>, <private dirty>, <private clean>, <swapPss dirty>, <rss total>, <heap size>, <heap alloc>, <heap free>]
+memoryParsers.Api30AndAbove = function Api30AndAbove (entries, valDict) {
```

Comment:
there is no need to create a separate namespace for this function. Simply define it being local to this module:

```
function parseMeminfoOutputForApi30Plus ...
```

Reply:
Also, a general agreement is that method names should always contain verbs